lib/std/special/build_runner.zig

```diff
@@ -331,3 +331,56 @@ fn argsRest(args: [][]const u8, idx: usize) ?[][]const u8 {
     if (idx >= args.len) return null;
     return args[idx..];
 }
+
+const progstr =
+    \\ pub fn main() u8 {
+    \\     return 1;
+    \\ }
+;
+
+const buildstr =
+    \\ const std = @import("std");
+    \\ pub fn build(b: *std.build.Builder) void {
+    \\     const exe = b.addExecutable("source", "source.zig");
+    \\     exe.install();
+    \\     const run_cmd = exe.run();
+    \\     run_cmd.step.dependOn(b.getInstallStep());
+    \\     const run_step = b.step("run", "Run");
+    \\     run_step.dependOn(&run_cmd.step);
+    \\ }
+;
+
+test "issue 10381" {
+    const testing = std.testing;
+    const allocator = testing.allocator;
+
+    var it = try std.process.argsWithAllocator(allocator);
+    const testargs = try testing.getTestArgs(&it);
+    defer it.deinit();
```

Comment:
This defer should be one line up, though currently it doesn't matter as this test isn't even being referenced. Maybe `std/build.zig` would be a better place for it?

Reply:
Oh no, that was an accident, the test is meant to be referenced. I have moved it to std.build and pushed the changes